apps/src/templates/progress/SendLesson.jsx

```diff
@@ -0,0 +1,57 @@
+import React from 'react';
+import PropTypes from 'prop-types';
+import i18n from '@cdo/locale';
+import Button from '@cdo/apps/templates/Button';
+import SendLessonDialog from './SendLessonDialog';
+
+export default class SendLesson extends React.Component {
+  static propTypes = {
+    lessonUrl: PropTypes.string.isRequired,
+    lessonTitle: PropTypes.string,
+    courseid: PropTypes.number,
+    analyticsData: PropTypes.string,
+    buttonStyle: PropTypes.object
+  };
+
+  constructor(props) {
+    super(props);
+    this.openDialog = this.openDialog.bind(this);
+    this.closeDialog = this.closeDialog.bind(this);
+    this.state = {
+      isDialogOpen: false
+    };
+  }
+
+  openDialog() {
+    this.setState({isDialogOpen: true});
+  }
+
+  closeDialog() {
+    this.setState({isDialogOpen: false});
+  }
+
+  render() {
+    return (
+      <div>
+        <Button
+          __useDeprecatedTag
```

Comment:
Is `__useDeprecatedTag` needed?

Reply:
Yes, unfortunately.  The Button renders differently depending on whether this tag is present and I needed the button to line up with the button above it (which has this tag).  I think we need to bite the bullet at some point and remove this from as many components as possible to really deprecate the legacy style.